SYCL/ESIMD/dpas/deprecated/dpas_tf32.cpp

```diff
@@ -0,0 +1,89 @@
+//==---------------- dpas_tf32.cpp  - DPC++ ESIMD on-device test ----------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu-intel-pvc || esimd_emulator
+// UNSUPPORTED: cuda || hip
+// RUN: %clangxx -fsycl -DESIMD_XE_HPC %s -o %t.out
```

Comment:
ESIMD_XE_HPC should not be needed

Reply:
Right, Thank you! I removed usage of those macro from tests here: 4240b1334a875eec376dbcd97cdbc3f47a89dfac
Also, adjusted paths to esimd_test_utils.hpp in the dpas/deprecated/* tests.